snuba/admin/static/clickhouse_queries/types.tsx

```diff
@@ -9,4 +9,27 @@ type ClickhouseNodeData = {
   local_nodes: ClickhouseNode[];
 };
 
-export { ClickhouseNodeData };
+type ClickhouseCannedQuery = {
+  description?: string;
+  name: string;
+  sql?: string;
```

Comment:
How come description and sql are optional, shouldn't we always get this from the API?

Reply:
description is not present for one of the predefined queries.

sql should always be, not sure why I did that

Reply:
It seems like the description key is actually always there but it is nullable: https://github.com/getsentry/snuba/blob/master/snuba/admin/clickhouse/system_queries.py#L54.
So the type should maybe be `string | null`